model/patch/db.go

```diff
@@ -175,6 +177,19 @@ func PatchesByProject(projectId string, ts time.Time, limit int) db.Q {
 	}).Sort([]string{"-" + CreateTimeKey}).Limit(limit)
 }
 
+func FindFailedCommitQueuePatchesinTimeRange(projectID string, startTime, endTime time.Time) db.Q {
```

Comment:
is there no reason to just return patches directly from this and bypass the db.Q

Reply:
No reason besides convention within the file. I don't mind changing it.